files/generate_vars.py

```diff
@@ -95,32 +93,37 @@ def _log_to_console(self, cmd, log):
         for line in iter(proc.stderr.readline, ''):
             log.error(line)
 
-    def _set_log(self, log_file, log_level):
+    @staticmethod
+    def _set_log(log_file, log_level):
         logger = logging.getLogger(PREFIX)
-        formatter = logging.Formatter(
-            '%(asctime)s %(levelname)s %(message)s')
-        if log_file is not None and log_file != '':
+
+        if log_file:
```

Comment:
I think that the verbose way is better, also you removed the check for empty string.

Reply:
Reverting in 2 places.
I thought the short way is more clear.
Python treats empty strings/lists/dict as false.
So instead of testing some string/list for not being None and then not being empty, you can just write "if obj".
But if it's confusing, I will revert.